parser/src/parser.rs

```diff
@@ -273,7 +273,7 @@ impl<'a> Parser<'a> {
 
     /// Parses the left-hand side of the next value.
     fn lhs(&mut self) -> ParseResult<Expr<'a>> {
-        self.repos("Expected value")?;
+        self.repos("Expected expression")?;
```

Comment:
```suggestion
        self.repos("Expected value")?;
```
This is `lhs()` for values, not `expression()`.